source/features/deep-reblame.tsx

```diff
@@ -60,22 +58,12 @@ async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchor
 }
 
 function addButton(pullRequest: HTMLElement): void {
-	const hunk = pullRequest.closest('.blame-hunk')!;
+	const hunk = pullRequest.closest('.react-blame-for-range')!;
 
-	const reblameLink = $('.reblame-link', hunk);
+	const reblameLink = $('.tooltipped-n', hunk);
 	if (reblameLink) {
 		reblameLink.setAttribute('aria-label', 'View blame prior to this change. Hold `Alt` to extract commits from this PR first');
 		reblameLink.classList.add('rgh-deep-reblame');
-	} else {
-		$('.blob-reblame', hunk)!.append(
-			<button
-				type="button"
-				aria-label="View blame prior to this change (extracts commits from this PR first)"
-				className="reblame-link btn-link no-underline tooltipped tooltipped-e d-inline-block pr-1 rgh-deep-reblame"
-			>
-				<VersionsIcon/>
-			</button>,
-		);
```

Comment:
?

This piece of code added the button to hunks that don't already have one.

For example the first PR commit in a file.

Test URL:

https://github.com/refined-github/refined-github/blame/main/source/github-helpers/heat-map.css

Every line here should have that button.